alkos/kernel/arch/x86_64/include/io.hpp

```diff
@@ -0,0 +1,34 @@
+#ifndef ARCH_X86_64_IO_HPP_
+#define ARCH_X86_64_IO_HPP_
+
+#include <defines.hpp>
+#include <types.hpp>
+
+WRAP_CALL byte inb(const u16 port)
```

Comment:
Ok. WRAP CALL.
Unless you envision some specific flags for this other than force inline, why not just use force inline